content/concept-amendments.md

```diff
@@ -86,13 +86,122 @@ When an amendment gets enabled for the network after the voting process, servers
+      "validation_quorum": 4,
+      "validator_list_expires": "2018-Jan-24 00:00:00"
+    },
+    "status": "success"
+  }
+}
+```
+
+If your server is not amendment blocked, the `amendment_blocked` field is not returned in the response.
+
+***TODO: Question: Update the `server_info` doc to include an `amendment_blocked` field descr? https://ripple.com/build/rippled-apis/#server-info. Does this amendment_blocked also display in `server_state`? https://ripple.com/build/rippled-apis/#server-state. If yes, also add descr there?***
+
+To find out which amendments are blocking your `rippled` server, use the [`feature`](reference-rippled.html#feature) admin command and look for features (amendments) that have `"enabled" : true` and `"supported" : false`. These values for a feature mean that the amendment is currently enabled (required) in the latest ledger, but your server does not know how to apply the amendment. For example:
+
+***TODO: Update `features` doc to talk about the implications of when `"enabled" : true` and `"supported" : false`. Useful? https://ripple.com/build/rippled-apis/#feature***
+
+```
+{
+   "id":1,
+   "result":{
+      "features":{
+         "B4D44CC3111ADD964E846FC57760C8B50FFCD5A82C86A72756F6B058DDDF96AD":{
+            "enabled":true,
+            "supported":false,
+            "vetoed":false
+         },
+         "6C92211186613F9647A89DFFBAB8F94C99D4C7E956D495270789128569177DA1":{
+            "enabled":true,
+            "supported":false,
+            "vetoed":false
+         },
+         "B9E739B8296B4A1BB29BE990B17D66E21B62A300A909F25AC55C22D6C72E1F9D":{
+            "enabled":true,
+            "supported":false,
+            "vetoed":false
+         },
+         "1D3463A5891F9E589C5AE839FFAC4A917CE96197098A1EF22304E1BC5B98A454":{
+            "enabled":true,
+            "supported":false,
+            "vetoed":false
+         },
+         "CC5ABAE4F3EC92E94A59B1908C2BE82D2228B6485C00AFF8F22DF930D89C194E":{
+            "enabled":true,
+            "supported":false,
+            "vetoed":false
+         },
+...
+         "08DE7D96082187F6E6578530258C77FAABABE4C20474BDB82F04B021F1A68647":{
+            "enabled":false,
+            "name":"PayChan",
+            "supported":true,
+            "vetoed":false
+         }
+      },
+      "status":"success"
+   }
+}
+```
+
+In this example, conflicts with the following features are causing the `rippled` server to be amendment blocked:
+
+* `B4D44CC3111ADD964E846FC57760C8B50FFCD5A82C86A72756F6B058DDDF96AD`
+
+* `6C92211186613F9647A89DFFBAB8F94C99D4C7E956D495270789128569177DA1`
+
+* `B9E739B8296B4A1BB29BE990B17D66E21B62A300A909F25AC55C22D6C72E1F9D`
+
+* `1D3463A5891F9E589C5AE839FFAC4A917CE96197098A1EF22304E1BC5B98A454`
+
+* `CC5ABAE4F3EC92E94A59B1908C2BE82D2228B6485C00AFF8F22DF930D89C194E`
+
+***TODO: Still need to build out an explanation of how these features became the ones that are amendment blocking the server. Working on amendment blocking my rippled server.***
```

Comment:
The typical case is you're running a version of `rippled` that's older than the one used by trusted validators.